### PR TITLE
Matrix

### PR DESCRIPTION
**Pull request by RajSpeed**
Hope you like it :)
matrix

### CODE REVIEW

File: matrix/404.html [hunk 1]
```diff
@@ -22,7 +22,10 @@ <h2>This page does not exist.</h2>
             src="https://buttons.github.io/buttons.js"
         ></script>
         <!-- partial -->
-        <script src="https://cdn.jsdelivr.net/npm/tsparticles@2.1.4/tsparticles.bundle.min.js"></script>
+        <script
+            type="text/javascript"
+            src="https://cdn.jsdelivr.net/npm/tsparticles@2.3.4/tsparticles.bundle.min.js"
+        ></script>
         <script src="https://cdn.jsdelivr.net/npm/tsparticles-path-polygon@2.1.4/tsparticles.path.polygon.min.js"></script>
```
Comment (matteobruni) [inline matrix/404.html:29]: This version is not updated, please use all tsParticles latest versions

**Review by matteobruni — CHANGES_REQUESTED** [review]

File: data/pages.json [hunk 2]
```diff
@@ -108,5 +108,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/tsparticles/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
+    },
+    {
+        "description":"Matrix 404",
+        "imageUrl":"https://raw.githubusercontent.com/RajSpeed/404-templates/matrix/__screenshots/matrix.png?raw=true",
```
Comment (matteobruni) [inline data/pages.json:114]: This URL is not correct, please use this repository as a source for images

Comment (RajSpeed) [reply data/pages.json]: Done😀

**Comment by matteobruni** [comment]
This is good, once you resolve the conflicts, it will be merged

**Review by matteobruni — APPROVED** [review]

**Pull request merged**